### DISCUSSION TITLE
js-ts-mode seems to override the nice xref-find-definitions keybinding

### DISCUSSION THREAD

**Discussion by trev-dev** [joaotavora/eglot#1348] [Q&A] [answered]
For whatever reason js-ts-mode comes with unwanted keybindings that take precedence over the standard xref defaults.  I'm asking about this here since other eglot users may have felt concerned about this and worked around it already.  So, how do I get js-ts-mode to not do that?  The `js-find-symbol` doesn't seem to work, I'd prefer to let the LSP server do its job.

![image](https://github.com/joaotavora/eglot/assets/28788713/5dd96de9-b7d9-4ab2-be15-60ce214ef2d5)

**Comment by trev-dev** [comment, marked as answer]
```elisp
(defun td-syntax-js-fixes ()
  (local-set-key (kbd "M-.") #'xref-find-definitions))
(add-hook 'js-ts-mode-hook 'td-syntax-js-fixes)
```

**Comment by joaotavora** [comment]
This is a bug for js-ts-mode. Please report it there

**Reply by trev-dev** [reply]
Sure thing.
